### ISSUE TITLE
[Feature request] Adguard mirror for cross site functional scripts

### ISSUE THREAD

**Issue by Itsnothectic** [AdguardTeam/AdguardForiOS#346]
I dont know if this would be legal, if it isnt then dont bother.

It would be nice to have adguard DNS host a mirror to googleapis.com and gstatic.com and all
the subdomains, mainly fonts and ajax.

Even with a VPN theres way too many browser identifiers that cant be blocked on iOS.

Google however takes this very personally, its against their ToS to use their services while interfering with their lifeblood however it is the webmasters and not the client that agrees to the terms for their phree(Im coining this now) APIs.

If this feature wont be implemented then could we point our blacklist to custom IPs? Firstly 0.0.0.0 would be faster than loopback, secondly this would give people the opportunity to run their own google APIs server.

If you like this idea then cloudfront is another company that uses functional tracking, by which I mean its widespread and can not be blocked on mobile due to it breaking websites such as many shopping sites.

For those reading this who want to know how to avoid this type of tracking, using the Tor browser is the answer, you dont have to use the Tor network you can use a VPN and this will provide a decent level of privacy. The webmaster will likely assume the VPN is a Tor exit node.

**Comment by ameshkov** [comment]
> It would be nice to have adguard DNS host a mirror to googleapis.com and gstatic.com and all the subdomains, mainly fonts and ajax.

All those domains are protected by HTTPS, we can't provide a mirror with a valid certificate for them.
